checkstyle/src/main/resources/vitro-checkstyle/checkstyle-suppressions.xml

```diff
@@ -3,6 +3,9 @@
     "-//Puppy Crawl//DTD Suppressions 1.1//EN"
     "http://www.puppycrawl.com/dtds/suppressions_1_1.dtd">
 <suppressions>
+  <!-- For Debug Only -->
```

Comment:
Alternatively, you can run:
```
mvn clean install -Dcheckstyle.skip
```
...as opposed to adding more to the suppressions file. Either way.